src/Tribe/Modules/Options.php

```diff
@@ -120,8 +119,24 @@ private function adjust_timezone( $event, $meta ) {
 			} else if ( $missing_event_details ) {
 				return $event;
 			} else {
+				// If there is a meridian and it's "pm" then adjust times.
+				if (
+					isset( $event['EventStartMeridian'] )
+					&& strtolower( $event['EventStartMeridian'] ) === 'pm'
+					&& $event['EventStartHour'] < 12
+				) {
+					$event['EventStartHour'] += 12;
+				}
+				if (
+					isset( $event['EventEndMeridian'] )
+					&& strtolower( $event['EventEndMeridian'] ) === 'pm'
+					&& $event['EventEndHour'] < 12
+				) {
+					$event['EventEndHour'] += 12;
+				}
```

Comment:
Use "Yoda Conditions" on these two lines:
```suggestion
				if (
					isset( $event['EventStartMeridian'] )
					&& 'pm' === strtolower( $event['EventStartMeridian'] )
					&& $event['EventStartHour'] < 12
				) {
					$event['EventStartHour'] += 12;
				}
				if (
					isset( $event['EventEndMeridian'] )
					&& 'pm' === strtolower( $event['EventEndMeridian'] )
					&& $event['EventEndHour'] < 12
				) {
					$event['EventEndHour'] += 12;
				}
```

Reply:
Using Yoda-conditions, we stopped, I thought. :chuckle: